integration_tests/http-errors.js

```diff
@@ -0,0 +1,42 @@
+const { datadog, sendDistributionMetric } = require("datadog-lambda-js");
+const https = require("https");
+
+const urls = ["httpstat.us/400", "httpstat.us/500"];
```

Comment:
The failure is caused by the badly formatted urls right?

Reply:
Good call, fixed the format and now we are invoking the functions correctly.